vrt/ci.js

```diff
@@ -333,6 +314,14 @@ function someSnapshotsWereUpdated() {
   return result;
 }
 
+function handleError() {
+  // Fail the CI job if an error propagates to this level
+  // Any non critical function should handle error and log a useful message
+  // Basically everything other than the test results and pushing a new branch is non critical (PR/Comments/Labels/Reviewers)
+  log(`The job has failed, but it is not a failure.`);
+  process.exit(1);
```

Comment:
you don't need this line, it will crash anyways

Reply:
not sure why but it does not fail the ci job. i tried removing the handler, but you get a `UnhandledPromiseRejectionWarning` and then i tried re-throwing the er in the handler but it still doesn't work.